IdentityCore/src/MSIDRequestContext.h

```diff
@@ -29,5 +29,10 @@
 - (NSString *)logComponent;
 - (NSString *)telemetryRequestId;
 - (NSDictionary *)appRequestMetadata;
+/**
+ Temporal property to disable Family Refresh Token. This will be removed in future, added to allow 1P apps to disable this feature themselves.
+ Enabled by default, also configured to be enabled/disabled remotely by Microsoft.
+ */
+@property (nonatomic, readwrite) BOOL disableFRT;
```

Comment:
Should we move out it from the "context" object into global config? Context is mainly used for logging/telemetry purposes.

Reply:
@antrix1989  MSALGlobalConfig looks a better place for this, but the problem is that `disableFRT` can be set by OneAuth/MSAL CPP, is there a global config equivalent in CommonCore?

Reply:
Good idea, thanks a lot. Updated to use the static var in `MSIDAccountCredentialCache`